rpm/libsilkit4.spec

```
@@ -1,13 +1,13 @@
 %define version_major 4
 %define version_minor 0
 %define version_patch 56
-%define version_suffix rc1
+%define version_suffix %{nil}
 
 
-%if "%{version_suffix}"
-%define silkit_version %{version_major}.%{version_minor}.%{version_patch}~%{version_suffix}
-%else
+%if "%{version_suffix}" == ""
 %define silkit_version %{version_major}.%{version_minor}.%{version_patch}
+%else
+%define silkit_version %{version_major}.%{version_minor}.%{version_patch}~%{version_suffix}
 %endif
 
 Name:       libsilkit4
```

Does this work? Would remove another place to change if we bump the major version.

```suggestion
Name:       libsilkit%{version_major}
```

---

The `Requires` field of the `devel` and `utils` packages could be handled similarly if supported.